Project.toml

```diff
@@ -3,46 +3,25 @@ uuid = "aad1d28c-1a3f-47da-806d-2ed76430c1d6"
 authors = ["Yaroslav Kharkov", "Katharine Hyatt"]
 version = "0.0.1"
 
-
 [deps]
 ArgParse = "c7e460c6-2fb9-53a9-8c5b-16f535851c63"
+Braket = "19504a0f-b47d-4348-9127-acc6cc69ef67"
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
+CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
 DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 ITensors = "9136182c-28ba-11e9-034c-db9fb085ebd5"
+InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
+Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Missings = "e1d29d7a-bbdc-5cf2-9ac0-f12de2c33e28"
+Parsers = "69de0a69-1ddd-5017-9359-2bf0b02dc9f0"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
-Braket = "19504a0f-b47d-4348-9127-acc6cc69ef67"
-CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
-Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
-InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
-Parsers = "69de0a69-1ddd-5017-9359-2bf0b02dc9f0"
-
-[compat]
```

Comment:
Why'd `[compat]` vanish?

Reply:
Bumping dependency versions, which were locked in older versions by the `[compat]`, eliminated a segfault error I was getting during basic testing. I could have investigated the cause for the segfault more deeply, but this seemed like an easier fix.

Reply:
In general it's good practice to keep at least some `[compat]` bounds, `Aqua.jl` will likely shriek about not having any. You could "simply" try setting `[compat]` to the latest versions of each direct dependency and see if that resolves things.